create/how_to/updating.md

```diff
@@ -313,7 +331,11 @@ curl -X GET \
   -o mydocuments.json
 ```
 
-### Step 4: Upload your data to the latest version of MeiliSearch
+### Step 4: Delete the database folder
+
+To delete the old MeiliSearch version, you need to delete data.ms. data.ms should be at the root of the MeiliSearch binary, unless you chose [another location](https://docs.meilisearch.com/reference/features/configuration.html#database-path).
```

Comment:
Same here

Reply:
And by the way, you can commit the changes @curquiza proposed, directly here from the github interfac
![Capture d’écran 2021-09-23 à 15 13 31](https://user-images.githubusercontent.com/48251481/134513506-422d0908-cfb2-422c-8e07-36719ff273ae.png)
e